controller/src/beerocks/master/son_management.cpp

```diff
@@ -2060,9 +2060,8 @@ void son_management::handle_bml_message(Socket *sd,
             }
         };
 
-        // TODO: replace with a list of configured clients read from controller-db
-        std::vector<sMacAddr> client_list;
-        if (!client_list.size()) {
+        std::vector<sMacAddr> client_list = database.get_clients_with_persistent_data_configured();
```

Comment:
`auto client_list = ...` (nitpick).

Reply:
fixed, was actually a remainder from previous PR that only declared it without really using it (that's why it was explicit and not `auto`)